java/runtime/src/main/java/org/ray/runtime/task/TaskExecutor.java

```diff
@@ -70,10 +70,10 @@ public TaskExecutor(AbstractRayRuntime runtime) {
 
     List<NativeRayObject> returnObjects = new ArrayList<>();
     ClassLoader oldLoader = Thread.currentThread().getContextClassLoader();
+    // Find the executable object.
+    RayFunction rayFunction = runtime.getFunctionManager()
```

Comment:
Is it possible that it can't find the method and results in an exception thrown here? Since you moved it out of the try-catch clause, this will make the worker process crash.

Reply:
If the method can't be found, this line will only return null, won't throw exceptions.

Reply:
OK. A `NullPointerException` would be thrown inside `try` block and caught by `catch` block. Then you access `rayFunction` again in `catch` block, again a new `NullPointerException` would be thrown.

Reply:
I see. But I think it's desirable to crash worker if `rayFunction` is null. because it indicates a system bug, not an app-level exception. I'll add a `checkNonNull` to make it more explicit.

Reply:
I see. Thanks!